src/Common/Services.fs

```diff
@@ -66,11 +66,11 @@ module String =
     /// converts a list of chars to a string
     let implode chars = System.String (List.toArray chars)
 
-    /// splits a string into non-empty substrings that were separated by the given predicate
-    let split (c:char) (s:string) =
-        let r = s.Split ([|c|], StringSplitOptions.RemoveEmptyEntries )
-        Array.toList r
-
+    /// splits a string into an array of strings that were separated by the given char
```

Comment:
The comment does not match the implementation. Anyway, I would suggest to remove  this function entirely as it rewraps the String method Split in a custom String module.